scene/gui/code_edit.cpp

```diff
@@ -2116,25 +2115,32 @@ String CodeEdit::get_text_for_symbol_lookup() {
 		return String();
 	}
 
-	StringBuilder lookup_text;
+	return get_text_with_cursor_char(line, col);
+}
+
+String CodeEdit::get_text_with_cursor_char(int p_line, int p_column) const {
+	int line = p_line;
+	int col = p_column;
```

Comment:
What are these for?

Reply:
They're never changed so should just use the arguments instead

Reply:
Even if they were changed it's pointless, unless you want to use original value after modifying.